pipelines/add_local_features.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "39e24685-678c-43f4-ad4d-5c63572de678",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Modules will be reloaded every time a cell is executed\n",
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "98bdf818-9377-4360-b8db-1fcada81af48",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
```

Comment:
Most of the inputs are unused. Remove them.

Reply:
Removed